documentation/wiki/ChangeWaves-Dev.md

```diff
@@ -25,21 +25,21 @@ The Process:
 
 ## Creating a Change Wave
 1. In the `Microsoft.Build` project, open `SharedUtilities\ChangeWaves.cs`.
-2. Add a const string to identify the new wave, following the format:
+2. Add a static readonly Version to identify the new wave, following the format:
 ```c#
-public const string Wave17_4 = "17.4";
+public static readonly Version Wave17_4 = "17.4";
```

Comment:
Not in this case. The visual studio doc page for this is more reflective of the `ChangeWaves.md` file rather than the dev file. No changes needed there.